proof_range.go

```diff
@@ -53,18 +67,32 @@ func (proof *RangeProof) StringIndented(indent string) string {
 		indent)
 }
 
-// Verify that a leaf is some value.
-// Does not assume that the proof itself is value.
-// For that, use Verify(root).
-func (proof *RangeProof) VerifyItem(i int, key, value []byte) error {
+// The index of the first leaf (of the whole tree).
+// Returns -1 if the proof is nil.
+func (proof *RangeProof) LeftIndex() int64 {
+	if proof == nil {
+		return -1
+	}
+	return proof.LeftPath.Index()
+}
+
+// `i` is the relative index within this RangeProof.
```

Comment:
Not sure we need to explain what `i` is here? It is not a parameter/argument nor do we return it. It looks like an implementation detail. Looks like this comment belongs inside of the method.

Reply:
Removed in the new PR.